### PR TITLE
GitHub-21754: Renaming the 'Viewing the registry’s contents' section

### LINKED ISSUE #21754: [enterprise-4.3] Edit suggested in file registry/accessing-the-registry.adoc
### Which section(s) is the issue in?
Viewing the registry’s contents

### What needs fixing?
The title is absolute misleading! It should be "See which pods registry is running!
But tbh ... viewing the registy's content is missing at all. Same for managing the registry's content. On OCP 3.x we had a nice webUI to the registry ... where is it?

Add the correct content for the title or change the title!

### PR DESCRIPTION
**Pull request by pneedle-rh**
Applies to master, branch/enterprise-4.3, branch/enterprise-4.4, branch/enterprise-4.5 and branch/enterprise-4.6.

This relates to [GitHub issue 21754](https://github.com/openshift/openshift-docs/issues/21754).

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please.

@vikram-redhat  are any redirects needed for this, since the filename has changed to reflect the title and the purpose of the module more accurately?

File: modules/registry-checking-the-status-of-registry-pods.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * registry/accessing-the-registry.adoc
+
+[id="checking-the-status-of-registry-pods_{context}"]
+= Checking the status of the registry Pods
+
+As a cluster administrator you can list the image registry Pods running in the `openshift-image-registry` project and check their status.
```
Comment (ahardin-rh) [inline modules/registry-checking-the-status-of-registry-pods.adoc:8]: As a cluster administrator, you can

File: modules/registry-checking-the-status-of-registry-pods.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the Pods in the `openshift-image-registry` project and view their status:
+
```
Comment (ahardin-rh) [inline modules/registry-checking-the-status-of-registry-pods.adoc:18]: Looks like an extra line between the step and the + before the code block

**Comment by ahardin-rh** [comment]
Just some minor comments. Looks great!

File: modules/registry-checking-the-status-of-registry-pods.adoc (hunk 1 above)
Comment (pneedle-rh) [reply modules/registry-checking-the-status-of-registry-pods.adoc]: Thank you for your review! I have now resolved this suggestion.

File: modules/registry-checking-the-status-of-registry-pods.adoc (hunk 2 above)
Comment (pneedle-rh) [reply modules/registry-checking-the-status-of-registry-pods.adoc]: Good catch, thank you! Resolved.

**Pull request merged**